src/convertToObject.js

```diff
@@ -6,7 +6,22 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  if (sourceString.length === 0) {
+    return {};
+  }
+
+  const stringToObject = sourceString
+    .split(';')
+    .map((value) => {
+      return value.split(':').map((spaces) => spaces.trim());
+    })
+    .reduce((acc, key) => {
+      acc[key[0]] = key[1];
```

Comment:
The variable name 'acc' could be more descriptive. Consider renaming it to 'styleObject' or another meaningful name that reflects its purpose.